browser/mocha/replace.js

```diff
@@ -27,27 +41,34 @@ extendInterfaces('replace', function(context, teardown) {
           // The DOM to replace is the result of calling the "original"
           // `instanceTemplate` implementation:
           var dom = originalInstanceTemplate.apply(this, arguments);
+          // creates an iterator of the node tree
```

Comment:
This comment doesn't add much information to the line of code. The only wood not int he code is "tree" :P